app/client/src/components/shared/ColumnChart.tsx

```diff
@@ -122,45 +176,53 @@ export default function StackedBarChart({
       },
       plotOptions: {
         column: {
-          stacking: 'normal',
+          groupPadding: histogram ? 0 : 0.2,
+          pointPadding: histogram ? 0 : 0.1,
+          stacking: histogram ? undefined : 'normal',
         },
       },
       series,
       subtitle: {
+        style: {
+          fontSize,
+        },
         text: subtitle,
       },
       title: {
         text: title,
       },
       tooltip: {
+        backgroundColor: 'rgba(247, 247, 247, 0.95)',
         borderColor: '#000000',
-        formatter: function () {
-          return (
-            this.points?.reduce((s: any, point: any) => {
-              return (
-                s +
-                `<tr>
-                <td style="color:${point.series.color};fontWeight:bold">
-                  ${point.series.name}: 
-                </td>
-                <td>${point.y?.toLocaleString()}</td>
-              </tr>`
-              );
-            }, `<b>${this.x}</b><table>`) + '</table>'
-          );
+        formatter: histogram
+          ? histogramTooltipFormatter(xUnit, yUnit)
+          : stackedTooltipFormatter(xUnit, yUnit),
```

Comment:
No changes are needed at this time.

In the future we may want to make this more generic in case we add in more column charts. Maybe we would just pass in the `formatter` function and other parameters that currently rely on the `histogram` parameter.